xen/xenops_types.ml

```diff
@@ -37,10 +37,10 @@ module Vgpu = struct
   type nvidia = {
     physical_pci_address: address option; (* unused; promoted to Vgpu.t *)
     config_file: string option;
-    virtual_pci_address: address option;
-    type_id: string;
```

Comment:
Either these should've been options or have defaults for backwards compatibility.